cime_config/testdefs/testmods_dirs/clm/FatesSetupParamBuild/shell_commands

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
```

Comment:
Another way this could be done, is to add several similar type of scripts into something like a ...

cime_config/usermods_dirs/_include

so it could be used by both testmods and user-mods as well as being a place where similar error checking type bash shell_command scripts could be put. If in one directory the Shell syntax of

``` shell
#!/bin/bash
. $script

```

would need to be used. There's some strangeness to how the include user-mod structure works, so this might be a good way to go. And everything in that directory could be something designed to run interactively as I've set this one up. And that might facilitate adding some automated testing around running these scripts.

Reply:
I'm leaving this as an include testmod for now. But, I'll start an issue about potential changes here. And I think the bash way might be the way to go. I'm not for certain though...

Resolving for now.